js/src/analysis/getL10nObject.js

```diff
@@ -7,5 +7,5 @@ import { get } from "lodash-es";
  */
 export default function getL10nObject() {
 	// Returns the metabox object from wpseoScriptData if that exists. Else; return null.
-	return get( window, "wpseoScriptData.metabox", null );
```

Comment:
There seem to be various places where this is used as an object without `null` checks.

Either all places where this is used should first check if this is an object before keys are accessed or the comment changed to refer to the old behavior.

Reply:
Chose to fix the documentation.